README.md

```diff
@@ -235,6 +236,31 @@ This way of deploying shards is useful when you want to manage KSM shards throug
 
 The downside of using an auto-sharded setup comes from the rollout strategy supported by `StatefulSet`s. When managed by a `StatefulSet`, pods are replaced one at a time with each pod first getting terminated and then recreated. Besides such rollouts being slower, they will also lead to short downtime for each shard. If a Prometheus scrape happens during a rollout, it can miss some of the metrics exported by kube-state-metrics.
 
+### Daemonset sharding for pod metrics
+
+For pod metrics, they can be sharded per node with the following flag:
+* `--nodename`
+
+Each kube-state-metrics pod uses FieldSelector (spec.nodeName) to watch/list pod metrics only on the same node.
+
+A daemonset kube-state-metrics example:
+```
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-state-metrics
+spec:
+  template:
+    spec:
+      containers:
+      - name: kube-state-metrics
+        args:
+        - --resources=pods
+        - --nodename=$(NODE_NAME)
```

Comment:
Can you include the downward API in here https://kubernetes.io/docs/concepts/workloads/pods/downward-api/ It's not clear where `$NODE_NAME` is coming from otherwise.

Reply:
Updated. NODE_NAME is  from:
```
        env:
        - name: NODE_NAME
          valueFrom:
            fieldRef:
              apiVersion: v1
              fieldPath: spec.nodeName

```